src/components/Requirements/SubRequirement.vue

```diff
@@ -106,15 +107,23 @@ import CompletedSubReqCourse from '@/components/Requirements/CompletedSubReqCour
 import IncompleteSubReqCourse from '@/components/Requirements/IncompleteSubReqCourse.vue';
 import DropDownArrow from '@/components/DropDownArrow.vue';
 
-import { SubReqCourseSlot, CrseInfo } from '@/requirements/types';
+import { CrseInfo } from '@/requirements/types';
```

Comment:
Just a little confused on my end. Isn't `CrseInfo` an imported type? How are we able to keep this import statement for `SubRequirement.vue` and remove its `Vue.component()` call, but we wanted to get rid of `CrseInfo` for `IncompleteSubReqCourse.vue`?

Reply:
The bug with the vuetr plugin is that imported type cannot appear in props.